composer.json

```diff
@@ -10,30 +10,26 @@
     }
   ],
   "require": {
-    "php": ">=7.1",
+    "php": ">=7.4",
```

Comment:
na tu 7.4 bacha - ve spoustě extractoru s tím byly spíš problémy

Reply:
ok, budem na to mysliet.